test/bot/deactivated.jl

```diff
@@ -0,0 +1,2 @@
+# include("../deps/SnoopCompile/precompile/precompile_TestPackage.jl")
```

Comment:
Rather than committing this to the project, it might be best to have the `"precompileActivator"` testset write it, most likely to some tempfile. That way, if the tests get interrupted between "activator" and "deactivator", the directories won't be left in an inconsistent state.